backend/controllers/retentionReasons.js

```diff
@@ -20,10 +20,12 @@ const retentionReasonsFactory = (elite2Api, logError) => {
       const agencies = await elite2Api.getAgencies(res.locals)
       const agency = agencies.find(a => a.agencyId === offenderDetails.agencyId).description
       const offenderUrl = getOffenderUrl(offenderNo)
+      const retentionReasons = await dataComplianceApi.getOffenderRetentionReasons(res.locals)
```

Comment:
would generally run those 3 queries in parallel through using `Promise.all()`.

Reply:
```js
let [first, second, third] = await Promise.all([
    someThingThatReturnsAPromise(),
    otherThingThatReturnsAPromise(),
    lastThingThatReturnsAPromise()
 ]);

Reply:
Cheers, done